python_cross_site_scripting/GrandmaEJ_Url_Shortner_python/us.py

```
@@ -0,0 +1,101 @@
+        return jsonify(response), 200
+
+    @app.route('/<short_id>', methods=['GET'])
+    def redirect_to_url(short_id):
+        data = load_data()
+        if short_id not in data or url_expired(data[short_id]["expiry_time"]):
+            return jsonify({"status": 404, "error": "error occurrence"}), 404
+
+        return redirect(data[short_id]["original_url"])
+
+def generate_unique_custom_id(data):
+    custom_id = ''.join(random.choices(string.ascii_lowercase + string.digits, k=6))
```

Ruff (0.13.3)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)